pontoon/base/static/js/fluent_interface.js

```diff
@@ -57,7 +59,7 @@ var Pontoon = (function (my) {
         }
 
         // Attributes
-        if (!attributes) {
+        if (!(attributes && attributes.length)) {
```

Comment:
`attributes` are now guaranteed to be an array. Checking the length should be sufficient.